src/stratis_cli/_errors.py

```diff
@@ -33,6 +33,31 @@ class StratisCliRuntimeError(StratisCliError):
     """
 
 
+# This indicates a bug and so coverage is not necessary as this error
+# is not handled.
+class StratisCliPropertyNotFoundError(StratisCliRuntimeError):  # pragma: no cover
+    """
+    Exception raised when a requested property from FetchProperties DBus interface
+    does not exist.
+    """
+
+    def __init__(self, type_name, prop_name):
+        """ Initializer.
+
+            :param str type_name: the name of the type that does not support this property name
```

Comment:
Please update docs and error string since using the interface name. Might as well change ```type_name``` to ```interface_name``` here as well.